src/views/Login.tsx

```diff
@@ -1,11 +1,135 @@
-import React from "react";
-import Button from "../components/Button";
+import React, {useState} from "react";
+import TextInput from "../components/TextInput";
+import axios from "axios";
+import Hero from "../assets/images/women-signup.svg"
+import { FcGoogle } from "react-icons/fc";
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import * as yup from "yup"
+import { Formik } from "formik";
+import { Link } from "react-router-dom";
+import { setAuthToken, setIsLoggedIn } from "../redux/authSlice";
+import { useDispatch } from "react-redux";
+
 
 const Login: React.FC = () => {
+
+   const notify = (message:string) => toast(message);
+
+   
+const dispatch = useDispatch();
+  const [isLoading, setIsLoading]= useState(false)
+
+  const emailPattern = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
+
+  const loginSchema = yup.object().shape({
+    email: yup
+      .string()
+      .required('Email is required')
+      .email('Please enter a valid email').matches(emailPattern,"Please enter a valid email"),
+      
+    password: yup
+      .string()
+      .required('Password is required')
+      .min(6, 'Password must contain at least 6 characters'),
+  });
+
+
+    const handleLogin = async (values: { email: string; password: string; }) => {
+      try {
+        setIsLoading(true);
+        const response = await axios({
+          method: "POST",
+          url: `http://localhost:8080/users/login`,
+          data: {
+            email: values.email,
+            password: values.password,
+          },
+        });
```

Comment:
```suggestion
        const response = await axios({
          method: "POST",
          url: `http://localhost:8080/users/login`,
          data: {
            email: values.email,
            password: values.password,
          },
        });
```
we can add reusable  util that initiate the axios instance and also use environment variable for the base url